src/CMakeLists.txt

```diff
@@ -51,11 +51,31 @@ if (${Dawn_FOUND})
   list(APPEND AMBER_SOURCES dawn_engine_config.cc)
 endif()
 
+if (${AMBER_ENABLE_DXC})
+  list(APPEND AMBER_SOURCES dxc_helper.cc)
+endif()
+
 add_library(libamber ${AMBER_SOURCES})
 amber_default_compile_options(libamber)
 target_include_directories(libamber PRIVATE "${CMAKE_BINARY_DIR}")
 set_target_properties(libamber PROPERTIES OUTPUT_NAME "amber")
 
+if (${AMBER_ENABLE_DXC})
+  target_include_directories(libamber PRIVATE
+    "${PROJECT_SOURCE_DIR}/third_party/dxc/include"
```

Comment:
Similarly, this could be `${LLVM_SOURCE_DIR}/include` and `${LLVM_BINARY_DIR}/include`